front/app/containers/Admin/dashboard/summary/charts/BarChartActiveUsersByTime.tsx

```diff
@@ -200,7 +201,9 @@ class BarChartActiveUsersByTime extends React.PureComponent<
     const { serie } = this.state;
 
     const noData =
-      !serie || serie.every((item) => isEmpty(item)) || serie.length <= 0;
+      isNilOrError(serie) ||
```

Comment:
An additional check was needed for these series variable in multiple chart components.